spec/factories/prisons.rb

```diff
@@ -1,17 +1,27 @@
 FactoryBot.define do
   CLOSED_PRISON_CODES = PrisonService.prison_codes.reject { |x|
-    PrisonService::OPEN_PRISON_CODES.include?(x) || PrisonService::ENGLISH_HUB_PRISON_CODES.include?(x)
+    PrisonService::OPEN_PRISON_CODES.include?(x) ||
+      PrisonService::ENGLISH_HUB_PRISON_CODES.include?(x) ||
+      PrisonService::WOMENS_PRISON_CODES.include?(x)
   }
   class Elite2Prison
-    attr_accessor :code, :name
+    delegate :code, :name, :womens?, to: :@prison
```

Comment:
Is this supposed to be `:gender` rather than `:womens?`? That's the name of the new field on the `PrisonInfo` Struct.

```suggestion
    delegate :code, :name, :gender, to: :@prison
```